packages/altspacevr-extras/src/videoPlayerManager.ts

```diff
@@ -41,11 +43,38 @@ export class VideoPlayerManager {
         this._RPC.cleanup();
     }
 
+    // private userJoined(user: User) {
+    //     if (Object.keys(this.videos).length > 0) {
+    //         const userRPC = new UserRPC(user);
+    //         for (const actorID in this.videos) {
+    //             if (this.videos.hasOwnProperty(actorID)) {
+    //                 userRPC.emit('VideoPlay',
+    //                     {
+    //                         parentId: actorID,
+    //                         URL: this.videos[actorID].url,
+    //                         startTime: this.videos[actorID].basisTime + Date.now() / 1000.0
+    //                     });
+    //             }
+    //         }
+    //     }
+    //     this.hasAnyUserJoined = true;
+    // }
+
     private userJoined(user: User) {
-        if (Object.keys(this.videos).length > 0) {
-            const userRPC = new UserRPC(user);
-            for (const actorID in this.videos) {
-                if (this.videos.hasOwnProperty(actorID)) {
+        this.hasAnyUserJoined = true;
+        const userRPC = new UserRPC(user);
+        for (const actorID in this.videos) {
+            if (this.videos.hasOwnProperty(actorID)) {
+                if (this.state === "show") {
```

Comment:
Any strings that do not need to be localized should be surrounded by single quotes.

Reply:
Also the same of the other uses of double quotes below.